API.md

```diff
@@ -214,21 +218,21 @@ __Example__
 ```cs
 try
 {
-	// Check whether 'my-bucketname' exists or not.
-	bool found = await minioClient.BucketExistsAsync(bucketName);
-	Console.Out.WriteLine("bucket-name was " + ((found == true) ? "found" : "not found"));
+   // Check whether 'my-bucketname' exists or not.
+   bool found = await minioClient.BucketExistsAsync(bucketName);
+   Console.Out.WriteLine("bucket-name was " + ((found == true) ? "found" : "not found"));
```

Comment:
``bucket-name exists`` and `bucket-name does not exist`` is better. Than `not found`